core/src/main/scala/kafka/server/ReplicaManager.scala

```diff
@@ -556,7 +556,7 @@ class ReplicaManager(val config: KafkaConfig,
           case e: Throwable =>
             BrokerTopicStats.getBrokerTopicStats(topic).failedFetchRequestRate.mark()
             BrokerTopicStats.getBrokerAllTopicsStats().failedFetchRequestRate.mark()
-            error("Error processing fetch operation on partition [%s,%d] offset %d".format(topic, partition, offset), e)
+            error("Error processing fetch operation on partition [%s,%d] offset %d".format(topic, partition, offset))
```

Comment:
I don't think you meant to change this back (you added the `e` to trunk in a separate PR).